### PR TITLE
Oracle Feeders

### PR DESCRIPTION
**Pull request by onnovisser**
### Description

- Add section for pool managers to add oracle feeders
- Add page where oracle feeders can update prices for external loans, for the pools they're allowed to do so.
- Remove max price variance from create loan page

[#1849](https://github.com/centrifuge/apps/issues/1849)
#1813 

### Approvals

- [ ] Dev

### Screenshots

image

image

image

### CODE REVIEW

File: centrifuge-app/src/pages/IssuerCreatePool/TrancheInput.tsx [hunk 1]
```diff
@@ -88,7 +97,7 @@ export const TrancheInput: React.FC<{ canRemove?: boolean; currency?: string; is
                   )}
                 </Field>
                 <FieldWithErrorMessage
-                  as={NumberInput}
```
Comment (sophialittlejohn) [inline centrifuge-app/src/pages/IssuerCreatePool/TrancheInput.tsx:91]: seems odd, are you planning on reverting?

File: centrifuge-js/src/modules/pools.ts [hunk 2]
```diff
@@ -2932,10 +2945,13 @@ export function getPoolsModule(inst: Centrifuge) {
                     Isin: pricingInfo.priceId.isin,
                     maturityDate: new Date(info.schedule.maturity.fixed.date * 1000).toISOString(),
                     maturityExtensionDays: info.schedule.maturity.fixed.extension / SEC_PER_DAY,
-                    oracle: oraclePrices[pricingInfo.priceId.isin] || {
-                      value: new CurrencyBalance(0, currency.decimals),
-                      timestamp: 0,
-                    },
+                    oracle: oraclePrices[pricingInfo.priceId.isin] || [
+                      {
+                        value: new CurrencyBalance(0, 18),
```
Comment (sophialittlejohn) [inline centrifuge-js/src/modules/pools.ts:2950]:
```suggestion
                        value: new CurrencyBalance(0, currency.decimals),
```

**Review by sophialittlejohn — COMMENTED** [review]
nice work!

File: centrifuge-app/src/pages/IssuerCreatePool/TrancheInput.tsx (hunk 1 above)
Comment (onnovisser) [reply centrifuge-app/src/pages/IssuerCreatePool/TrancheInput.tsx]: fixed

**Review by sophialittlejohn — APPROVED** [review]

**Pull request merged**